tests/integration/multimodal/yaml/embedding-multimodal-concatenate.py

```diff
@@ -2,21 +2,16 @@
 __license__ = "Apache-2.0"
 
 import numpy as np
-from jina.executors.decorators import batching, as_ndarray
 from jina.executors.encoders.multimodal import BaseMultiModalEncoder
 
 
 class ConcatenateMultiModalEncoder(BaseMultiModalEncoder):
 
-    batch_size = 10
-
     def __init__(self,
                  *args,
                  **kwargs):
         super().__init__(*args, **kwargs)
 
-    #@batching
-    @as_ndarray
```

Comment:
Leep them here as they will be used in the future

Reply:
They did not make sense in the parent class but do here

Reply:
ok